src/pages/downloads.tsx

```diff
@@ -0,0 +1,225 @@
+          releaseNotesURL={''}
+          sourceCodeURL={'https://github.com/ethereum/go-ethereum/archive/v1.10.25.tar.gz'}
+          windowsBuildURL={'https://gethstore.blob.core.windows.net/builds/geth-windows-amd64-1.10.25-69568c55.exe'}
+        />
+
+        <DownloadsSection
+          imgSrc='/images/pages/gopher-home-side-desktop.svg'
+          imgAltText='Gopher facing right'
+          sectionTitle='Specific Versions'
+          id='specificversions'
+        >
+          <Stack p={4}>
+            <Text textStyle='quick-link-text'>
+              If you&apos;re looking for a specific release, operating system or architecture, below you will find:
+            </Text>
+
+            <UnorderedList px={4}>
+              <ListItem>
+                <Text textStyle='quick-link-text'>
+                  All stable and develop builds of Geth and tools
+                </Text>
+              </ListItem>
+              <ListItem>
+                <Text textStyle='quick-link-text'>
+                  Archives for non-primary processor architectures
+                </Text>
+              </ListItem>
+              <ListItem>
+                <Text textStyle='quick-link-text'>
+                  Android library archives and iOS XCode frameworks
+                </Text>
+              </ListItem>
+            </UnorderedList>
+
+            <Text textStyle='quick-link-text'>
+              Please select your desired platform from the lists below and download your bundle of choice. Please be aware that the MD5 checksums are provided by our binary hosting platform (Azure Blobstore) to help check for download errors. For security guarantees please verify any downloads via the attached PGP signature files (see{' '}
+              <Link
+                href={'#pgpsignatures'}
+                variant='light'
+              >
+                OpenPGP
+              </Link>{' '}
+              Signatures for details).
+            </Text>
+          </Stack>
+        </DownloadsSection>
+
+        <DownloadsSection sectionTitle='Stable releases' id='stablereleases'>
+          <Stack p={4} borderBottom='2px solid' borderColor='brand.light.primary'>
+            <Text textStyle='quick-link-text'>
+              These are the current and previous stable releases of go-ethereum, updated automatically when a new version is tagged in our{' '}
+              <Link
+                href={GETH_REPO_URL}
+                isExternal
+                variant='light'
+              >
+                GitHub repository.
+              </Link>
+            </Text>
+          </Stack>
+
+          {/* TODO: swap test data for real data */}
+          <DownloadsTable data={testDownloadData.slice(0, amountStableReleases)}/>
+
+          <Stack sx={{ mt: '0 !important' }}>
+            <Link as='button' variant='button-link-secondary' onClick={() => {
```

Comment:
onClick method should be moved to a named function for more clarity, something like

```
onClick={showMoreStableBuilds}
```